#! /bin/sh -e

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p0 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p0 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac
exit 0

--- capplets/default-applications/gnome-default-applications-properties-structs.c.orig	2003-10-18 16:10:51.000000000 +0200
+++ capplets/default-applications/gnome-default-applications-properties-structs.c	2003-10-18 16:12:44.000000000 +0200
@@ -36,6 +36,7 @@
 
 BrowserDescription possible_browsers[] =
 {
+        { N_("Debian Sensible Browser"), 			"sensible-browser",    "sensible-browser %s",    FALSE, FALSE, FALSE },
         { N_("Epiphany"), 			"epiphany",    "epiphany %s",    FALSE, FALSE, FALSE },
         { N_("Galeon"), 			"galeon",    "galeon %s",    FALSE, FALSE, FALSE },
         { N_("Encompass"), 			"encompass", "encompass %s", FALSE, FALSE, FALSE },
@@ -63,6 +64,7 @@
 
 TerminalDescription possible_terminals[] = 
 { 
+        { N_("Debian Terminal Emulator"), "x-terminal-emulator", "-e", FALSE },
         { N_("Gnome Terminal"), "gnome-terminal", "-x", FALSE },
         { N_("Standard XTerminal"), "xterm", "-e", FALSE },
         { N_("NXterm"), "nxterm", "-e", FALSE },
